doc/building_sdk.md

```diff
@@ -7,36 +9,22 @@ System Dependencies:
  - (Mac only) Xcode https://developer.apple.com/xcode
  - git https://git-scm.com
  - Docker https://www.docker.com (for local testing)
+ - JAVA_HOME environment variable set to JDK installation path
+ - Apache Ant http://ant.apache.org
 
 Get the SDK:
 
     git clone https://github.com/kbase/kb_sdk
+    git clone https://github.com/kbase/jars
 
 Pull dependencies and configure the SDK:
 
     cd kb_sdk
-    make bin
```

Comment:
I don't see a use case for this anymore, if a user wants to use the SDK without compiling just use the docker image